lms/templates/sysadmin_dashboard.html

```diff
@@ -7,6 +7,9 @@
   <%static:css group='style-course'/>
   <script type="text/javascript" src="${static.url('js/vendor/flot/jquery.flot.js')}"></script>
   <script type="text/javascript" src="${static.url('js/vendor/flot/jquery.flot.axislabels.js')}"></script>
+  %if modeflag.get('mgmt_commands'):
```

Comment:
define a new variable: `mgmt_command_enabled = modeflag.get('mgmt_commands')`

then each instance of this uses the same `get()` command.